app/assets/javascripts/find-data.js

```diff
@@ -0,0 +1,13 @@
+function sortDatasets() {
+    document.location.href = newLocation()
+
+    function newLocation() {
+        var currentLocation = document.location.href
+        var sortParam = "&sort="
```

Comment:
I recommend (and probably eventually mandate) to run all js files against jslint before committing.